### PR TITLE
Update missing/wrong information

### PR DESCRIPTION
**Pull request by Ente3**
## Type of change

- [ ] Add new directory for a server
- [X] Update directory of a server
- [ ] Remove directory of a server
- [X] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/baxnetzwerk/manifest.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "server_name": "Bax",
+    "nice_name": "Bax Netzwerk",
+    "direct_ip": "play-bax.de"
```
Comment (maaaathis) [inline minecraft_servers/baxnetzwerk/manifest.json:4]:
```suggestion
    "direct_ip": "play-bax.de",
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by maaaathis** [comment]
You don't need to open a new PR everytime btw.

**Comment by Ente3** [comment]
Oh okay, I committed suggested changes

File: minecraft_servers/baxnetzwerk/manifest.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "server_name": "BaxNetzwerk",
```
Comment (maaaathis) [inline minecraft_servers/baxnetzwerk/manifest.json:2]:
```suggestion
    "server_name": "baxnetzwerk",
```

**Review by maaaathis — CHANGES_REQUESTED** [review]
sry, missed that before

**Comment by Ente3** [comment]
Okay, everything resolved. Autocheck passes ^^

**Review by maaaathis — APPROVED** [review]
lgtm

**Comment by Ente3** [comment]
Hi, how to merge do master?
Merging still blocked

**Pull request merged**